data_model/src/events/data/events.rs

```diff
@@ -559,6 +559,65 @@ mod trigger {
     }
 }
 
+mod config {
+    use super::*;
+
+    /// Config Event
+    #[derive(
+        Clone,
+        PartialEq,
+        Eq,
+        PartialOrd,
+        Ord,
+        Hash,
+        Debug,
+        Decode,
+        Encode,
+        Deserialize,
+        Serialize,
+        IntoSchema,
+        Filter,
+    )]
+    #[non_exhaustive]
+    #[allow(missing_docs)]
+    pub enum ConfigurationEvent {
+        Registered(ParameterId),
+        Changed(ParameterId),
+        Unregistered(ParameterId),
```

Comment:
Do you plan to add support for parameter registration/unregistration? I can't find then used anywhere.

Reply:
We discussed some time ago with @appetrosyan that parameters should rather follow the versioning system, so this has to be removed.